dev/com.ibm.ws.microprofile.config_fat/bnd.bnd

```diff
@@ -26,8 +26,10 @@ src: \
 	test-applications/testAppUtils.jar/src,\
 	test-applications/sharedLib.jar/src,\
 	test-applications/defaultSources.jar/src,\
+	test-applications/cdiConfig.war/resources,\
```

Comment:
why are resource folders being added to build path?

Reply:
I can take them out.  They were in the eclipse build source dirs.